packages/cli/src/commands/redis/info.ts

```diff
@@ -13,7 +13,7 @@ export default class Info extends Command {
   }
 
   static args = {
-    database: Args.string(),
+    database: Args.string({description: 'Name of the Redis database. If omitted, it will default to the primary Redis instance associated with the app.'}),
```

Comment:
Change to "name of the Key-Value Store database. If omitted, it defaults to the primary instance associated with the app."